dvc.yaml

```diff
@@ -0,0 +1,132 @@
+stages:
+  preprocess_wellcome_science:
+    cmd: grants_tagger preprocess wellcome-science data/raw/science_tags_full_version.xlsx
+      data/processed/science_grants_tagged_title_synopsis.jsonl
+    deps:
+    - data/raw/science_tags_full_version.xlsx
+    - grants_tagger/preprocess.py
+    params:
+    - preprocess_wellcome_science.text_cols
+    - preprocess_wellcome_science.meta_cols
+    outs:
+    - data/processed/science_grants_tagged_title_synopsis.jsonl
+  train_tfidf_svm:
+    cmd: grants_tagger train data/processed/science_grants_tagged_title_synopsis.jsonl
+      models/label_binarizer-2020.05.2.pkl models/tfidf-svm-2020.05.2.pkl --approach tfidf-svm
+    deps:
+    - data/processed/science_grants_tagged_title_synopsis.jsonl
+    - grants_tagger/train.py
```

Comment:
I'm very late to this game, but just noting that if you change any of the upstream files that this depend on (for example `utils` and `.yield_data`), dvc won't know it unless you explicitly put it is as a dependency.

Reply:
This is a true, I think there is a fine balance to how much you add as dependencies so that you do not have to rerun repro every time but adding the utils is sensible.